src/thirdparty/bcd/bcd/ImageIO.cpp

```diff
@@ -21,11 +21,20 @@
     #pragma warning (disable: 4800)     // 'int': forcing value to bool 'true' or 'false' (performance warning)
 #endif
 
+#if __GNUC__ >= 7
```

Comment:
Since this is not a part of src/appleseed, it is okay to leave it like this without breaking the code consistency.

Reply:
So no need to put it to separate headers =)